.github/workflows/build_and_deploy.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build and Deploy
+on:
+  # Nightly build
+  schedule:
+    - cron: '0 9 * * *'
+  # Manual nightly & release
+  workflow_dispatch:
+    inputs:
+      mode:
+        description: What type of build to trigger. Release builds should be ran from the `master` branch.
+        required: true
+        default: nightly
+        type: choice
+        options:
+          - nightly
+          - release
+      macos:
+        description: Whether to build macOS
+        required: true
+        type: boolean
+        default: true
+      deploy:
+        description: Whether to deploy artifacts
+        required: true
+        type: boolean
+        default: true
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+env:
+  # XXX: UPDATE THIS BEFORE WHEN GOING LIVE
```

Comment:
Obviously this needs updating at some point.

Reply:
Yeah, this isn't going live today, it'll go live when all 3 platforms are ready and the mac mini can be decommissioned. The builds from here will be going to a separate bucket (not mapped to packages.element.io) for extra testing